docs/source/restore/_common/restore-raft-schema-warn.rst

```diff
@@ -0,0 +1 @@
+.. warning:: Restoring schema into a cluster with ScyllaDB **5.4.X** or **2024.X** with ``consistent_cluster_management: true`` isn't supported. Please see the following :ref:`workaround <restore-schema-workaround>`.
```

Comment:
@tzach @karol-kokoszka I changed restore schema safety validation so that:
- it always works with Scylla version < 5.4 / 2024.1
- i works when `consistent_cluster_management: false` and Scylla version 5.4 / 2024.1
- it newer works with Scylla version >= 6.0 / 2024.2

Reply:
> I think it should be 2024.1.x. I believe consistent_cluster_management: true will not be configurable for versions later than 5.4 and 2024.1 (it was already removed from the docs and scylla.yaml).

@annastuchlik it means that there will be no option to disable raft in new versions of Scylla, right ?